### PR TITLE
new(ci): add a release-body CI for drivers releases.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

This PR adds a new pipeline to generate github release body for `driver` tags.
The release-body is autogenerated from latest tag diff; moreover, [kernel testing matrixes](https://falcosecurity.github.io/libs/matrix/) are prepended, so that each driver release is quickly linked to its kernel testing matrix results.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/kernel_tests.yaml [hunk 1]
```diff
@@ -11,9 +11,11 @@ on:
   push:
     branches:
       - master
+    tags:
```
Comment (FedeDP) [inline .github/workflows/kernel_tests.yaml:14]: Run also on drivers tags.

File: .github/workflows/kernel_tests.yaml [hunk 2]
```diff
@@ -34,7 +36,7 @@ jobs:
         working-directory: ./ansible-playbooks
         run: |
           LIBS_V=${{ github.event.inputs.libsversion }}
-          LIBS_VERSION=${LIBS_V:-"master"}
+          LIBS_VERSION=${LIBS_V:-${{ github.ref_name }}}
```
Comment (FedeDP) [inline .github/workflows/kernel_tests.yaml:39]: Use `ref_name` as default libs version, that is either `master` or the tag name.

File: .github/workflows/release-body.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+name: Generate release body
+on:
+  workflow_run:
+    workflows: ["Test drivers against a matrix of kernels/distros"]
+    types: [completed]
+    branches-ignore: [master] # only when it runs on tags
```
Comment (FedeDP) [inline .github/workflows/release-body.yml:6]: Do not trigger on master completions.

File: .github/workflows/release-body.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      
+permissions:
```
Comment (FedeDP) [inline .github/workflows/release-body.yml:8]: Needed to write release body.

File: .github/workflows/release-body.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+          sed -i 's/\[\(.\)\]([^)]*)/\1/g' matrix_ARM64.md
+          sed -i '1s/^/# Driver Testing Matrix amd64\n\n/' matrix_X64.md
+          sed -i '1s/^/# Driver Testing Matrix arm64\n\n/' matrix_ARM64.md
+          cat matrix_X64.md matrix_ARM64.md > release-body.md
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          body_path: ./release-body.md
+          generate_release_notes: true
```
Comment (FedeDP) [inline .github/workflows/release-body.yml:52]: >  If body is specified, the body will be pre-pended to the automatically generated notes

https://github.com/softprops/action-gh-release#inputs

File: .github/workflows/kernel_tests.yaml [hunk 6]
```diff
@@ -11,9 +11,11 @@ on:
   push:
     branches:
       - master
+    tags:
+      - '*+driver'
```
Comment (FedeDP) [inline .github/workflows/kernel_tests.yaml:15]: Needs to be fixed :/

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by jasondellaluce — APPROVED** [review]

**Comment by FedeDP** [comment]
This needs to be fully tested, of course, but should work just fine (and, after all, it is not a release blocking CI, therefore none of value would be lost if it breaks!)

**Comment by FedeDP** [comment]
Ok master CI is passed and `release-body` worfklow did not trigger: https://github.com/falcosecurity/libs/actions/workflows/release-body.yml
